dist/index.html

```diff
@@ -18,7 +18,8 @@
   <!--<meta name="smartbanner:api" content="true">-->
   <!--<meta name="smartbanner:hide-ttl" content="2629746000">-->
   <!--<meta name="smartbanner:hide-path" content="/">-->
-  <!--<meta name="smartbanner:disable-positioning" content="true">-->
+  <meta name="smartbanner:disable-positioning" content="true">
+  <meta name="smartbanner:parent-element" content="#container">
```

Comment:
Let's hook it up with your example in the README:
```suggestion
  <meta name="smartbanner:parent-element" content=".banner-container">
```